src/main/kotlin/net/emdal/tank/clause/WhereClause.kt

```diff
@@ -0,0 +1,24 @@
+package net.emdal.tank.clause
+
+import net.emdal.tank.Entity
+import net.emdal.tank.IntProperty
+import net.emdal.tank.Property
+import net.emdal.tank.StringProperty
+
+class WhereClause(
+  private val alias: String,
+  override var query: List<String> = listOf("WHERE "),
+  override var aliases: Map<String, List<Entity>> = emptyMap()
+) : Clause {
+  infix fun Property.greaterThan(value: Any) = WhereClause(

```

Comment:
Should greater than be allowed for all `Property`s. Might not make sense for string or other types of properties. This should be a function of `IntProperty`.